internal/cluster/sbd.go

```diff
@@ -86,23 +100,31 @@ func NewSBD(executor utils.CommandExecutor, cluster, sbdPath, sbdConfigPath stri
 	return s, nil
 }
 
-func getSBDConfig(sbdConfigPath string) (map[string]interface{}, error) {
-	sbdConfFile, err := os.Open(sbdConfigPath)
+func LoadSbdConfig(sbdConfigPath string) (map[string]string, error) {
+	sbdConfigFile, err := os.Open(sbdConfigPath)
 	if err != nil {
-		return nil, errors.Wrap(err, "could not open sbd config file")
+		return nil, &SBDConfigLoadingError{
+			Type:  SDBConfigReadingError,
+			inner: errors.Wrap(err, "could not open sbd config file"),
+		}
 	}
 
-	defer sbdConfFile.Close()
-
-	sbdConfigRaw, err := io.ReadAll(sbdConfFile)
+	defer func() {
+		err := sbdConfigFile.Close()
+		if err != nil {
+			log.Error(err)
+		}
+	}()
 
+	conf, err := envparse.Parse(sbdConfigFile)
 	if err != nil {
-		return nil, errors.Wrap(err, "could not read sbd config file")
+		return nil, &SBDConfigLoadingError{
+			Type:  SDBConfigDecodingError,
+			inner: errors.Wrap(err, "could not read sbd config file"),
+		}
```

Comment:
in case we could remove the extra entries so the output is the same to what we had with the regex

Reply:
At the end I returned to my first proposal of not skipping empty values in config and covered in test.

If this is sorted out we need to agree on https://github.com/trento-project/web/pull/996.